src/applications/ivc-champva/10-7959a/chapters/healthInsuranceInformation.js

```diff
@@ -0,0 +1,231 @@
+import get from 'platform/utilities/data/get';
+import { VaTextInputField } from 'platform/forms-system/src/js/web-component-fields';
+import React from 'react';
+import {
+  titleUI,
+  titleSchema,
+  yesNoUI,
+  yesNoSchema,
+  arrayBuilderItemFirstPageTitleUI,
+  arrayBuilderItemSubsequentPageTitleUI,
+  arrayBuilderYesNoSchema,
+  arrayBuilderYesNoUI,
+  phoneUI,
+  phoneSchema,
+  textUI,
+  textSchema,
+  radioUI,
+  radioSchema,
+} from 'platform/forms-system/src/js/web-component-patterns';
+import { arrayBuilderPages } from 'platform/forms-system/src/js/patterns/array-builder';
+import { nameWording } from '../../shared/utilities';
+
+const radioLabels = {
+  group: 'Employer sponsored insurance (group)',
+  nonGroup: 'Individual or private insurance (non-group)',
+  medicare: 'Medicare Part A or B (hospital and medical insurance)',
+  other: 'Insurance type not listed',
+};
+
+const hintText =
+  'This includes any coverage through a family member, health insurance from an employer, and supplemental or secondary insurance.';
+
+const yesNoContent = {
+  title:
+    'Does the beneficiary have any other health insurance coverage that isn’t through CHAMPVA?',
+  labels: {
+    Y: 'Yes',
+    N: 'No',
+  },
+  hint: hintText,
+  labelHeaderLevel: '5',
+};
+
+/** @type {ArrayBuilderOptions} */
+const options = {
+  arrayPath: 'policies',
+  nounSingular: 'policy',
+  nounPlural: 'policies',
+  required: true,
+  isItemIncomplete: item => !item?.name && !item?.type, // include all required fields here
+  maxItems: 2,
+  text: {
+    getItemName: item => item.name,
+    cardDescription: item =>
+      item?.type === 'other'
+        ? `${item?.otherType}`
+        : `${radioLabels[(item?.type)]}`,
+    summaryTitle: 'Beneficiary’s health insurance review',
+  },
+};
+
+export const insuranceStatusSchema = {
```

Comment:
Standalone page that asks the user if they have OHI to add. If yes, they enter the list loop on next screen. Otherwise they get booted to the next section.